helix-term/src/commands.rs

```diff
@@ -4143,6 +4167,18 @@ pub mod insert {
         helix_event::dispatch(PostInsertChar { c, cx });
     }
 
+    pub fn insert_string(cx: &mut Context, string: String) {
```

Comment:
Can we use `&str` here to skip an allocation? `Tendril:from()` ends up using `Tendril::from_slice(&*input)`